packages/components/psammead-topic-tags/src/index.stories.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
+import { TopicTags, TopicTag } from '.';
+import notes from '../README.md';
+
+const STORY_KIND = 'Components/TopicTags';
+
+storiesOf(STORY_KIND, module)
+  .addDecorator(withKnobs)
+  .addDecorator(withServicesKnob())
+  .add(
+    'default',
+    ({ dir, service, script, text }) => {
+      const shortText = service === 'news' ? text : text.trim().split(' ')[0];
+
+      return (
+        <TopicTags dir={dir} service={service} script={script}>
```

Comment:
Is there a reason `dir` is set here and in other places? Unless we are using the value (for example in styles, like `script`), the direction can be inferred by the global document direction, which is set at the root of every World Service page. See the root element of https://www.bbc.com/arabic.